packages/screens/AdministrationDashboard/component/CurrentlyHighLightedProject.tsx

```diff
@@ -0,0 +1,291 @@
+                                    height: 28,
+                                  }}
+                                  source={avaPNG}
+                                />
+                                <BrandText
+                                  style={[
+                                    fontSemibold16,
+                                    { marginLeft: 15, marginRight: 10 },
+                                  ]}
+                                >
+                                  Meebits
+                                </BrandText>
+                                <SVG source={checkBadgeSVG} />
+                              </View>
+                            )}
+                            keyExtractor={(item: any) => item.id}
+                            style={{
+                              marginTop: 12,
+                              marginLeft: 5,
+                            }}
+                            contentContainerStyle={{
+                              height: 180,
+                            }}
+                          />
+                        </View>
+                      </Box>
+                    ) : (
+                      <ImageBackground
+                        source={guardianPng}
+                        imageStyle={{ borderRadius: 16 }}
+                        style={styles.imageBgStyle}
+                      >
+                        <TouchableOpacity
+                          onPress={() => {
+                            const old = [...guardiansHandler];
+                            old.splice(index, 1);
+                            setGuardiansHandler(old);
+                          }}
+                          style={styles.trashBtnBox}
+                        >
+                          <SVG source={trashSVG} />
+                        </TouchableOpacity>
+                      </ImageBackground>
+                    )}
+                  </Box>
+                </View>
+              </>
+            );
+          })}
+          {guardiansHandler?.length < 5 ? (
+            <Box
+              notched
+              style={[
+                styles.herosLisBox,
+                {
+                  width: width >= MD_BREAKPOINT ? "19%" : "100%",
+                  marginTop: 52,
+                },
+              ]}
+            >
+              <TouchableOpacity
+                onPress={() => setGuardiansHandler([...guardiansHandler, 1])}
+                style={styles.plusBox}
+              >
+                <View style={styles.dotBackground}>
+                  <SVG source={addCircleSVG} />
+                </View>
+                <BrandText style={[fontSemibold14, { marginLeft: 5 }]}>
+                  Add more
+                </BrandText>
+              </TouchableOpacity>
+            </Box>
+          ) : null}
+        </View>
+      </View>
+    </View>
+  );
+};
+
+// eslint-disable-next-line no-restricted-syntax
+const styles = StyleSheet.create({
```

Comment:
We try to not use `StyleSheet.create` so you have an example here https://github.com/TERITORI/teritori-dapp/blob/main/packages/components/music/FeedMusicList.tsx on what we want to use styles now 👍

Reply:
Fixed, PR updated